yo/static_content.py

```diff
@@ -0,0 +1,61 @@
+from aiohttp import web
```

Comment:
Is there a reason to use python for serving static content instead of nginx, s3, etc?